### PR TITLE
Updating to next development version

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Update to the next dev version of 2.12.0-SNAPSHOT
Fixed a doc dependency issue.
#### How should this be tested? (List steps with links to updated documentation)
Successful ci build

### CODE REVIEW

**Comment by clockard** [comment]
build now

File: distribution/ddf/pom.xml [hunk 1]
```diff
@@ -258,6 +258,11 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
+            <groupId>ddf</groupId>
```
Comment (shaundmorris) [inline distribution/ddf/pom.xml]: ❓ Was this discovered when you bumped the version?

Comment (clockard) [reply distribution/ddf/pom.xml]: Yes. In a multi-threaded build the ddf distribution was getting built in parallel with the docs.

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Pull request merged**